package.json

```diff
@@ -36,6 +36,7 @@
     "express-session": "^1.15.6",
     "faker": "^4.1.0",
     "jsonwebtoken": "^8.3.0",
+    "jwt-simple": "^0.5.1",
```

Comment:
I didn't see where this is used, if it's not used in the code, please remove

Reply:
done.